### DISCUSSION TITLE
keyboardCallback in ConstrantDemo

### DISCUSSION THREAD

**Discussion by Neo-X** [bulletphysics/bullet3#3492] [General]
Hello,

I think in the constraint demo the keyboardCallback(int key, int x, int y) function is meant to override CommonRigidBodyBase::keyboardCallback(int key, int state).

This change will allow the keyboardCallback in the Constraint demo to be called properly through OpenGLExampleBrowser::MyKeyboardCallback(int key, int state).

**Comment by benelot** [comment]
You could make a pull request based on your finding to fix the issue. Would you like to do that?

**Comment by Neo-X** [comment]
I will see what I can do. I have made so many other changes I am not sure the pull would go well at this point.
